### ISSUE TITLE
Response delay not working?

### ISSUE THREAD

**Issue by mitchconquer** [mswjs/msw#136]
## Describe the bug
Hi! I'm using MSW with Create React App Typescript and I'm having an issue getting the response delay to work. The MSW log message in the console _does_ respect the response delay but the fetch response does not.

## Environment

- `msw: ^0.13.1`
- `nodejs: v13.12.0`
- `npm: 6.14.4`
- `react-scripts: 3.4.1`
- `typescript: ^3.8.3`
- Chrome Version 81.0.4044.129 (Official Build) (64-bit)

## To Reproduce

I have created a [code sandbox to try to reproduce this issue](https://codesandbox.io/s/msw-issue-delay-not-working-sd0ys?file=/src/localApiMock.js:444-502) but cannot get it to work. Not sure if you have any guidance on that?

1. Create CRA with `--template typescript`
2. Add MSW with `yarn add -D msw`
3. Run `npx msw init ./public`
4. Create route with delay
5. Fetch to route

When fetch to route, the response comes back immediately but the MSW console message waits for the delay and then appears.

## Expected behavior
I expect the response from the mock API to wait until the delay before returning. Is that a misunderstanding of the library?

## Screenshots
I have created a GIF to show the issue. I put a console log around my fetch statement like below to show when the request starts and finishes.

```javascript
console.log(`Fetch request started: ${new Date()}`);

const res = await window.fetch(
  `${API_URL_BASE}/${API_PREFIX}/${endpoint}`,
  config
);

  console.log(`Fetch response received: ${new Date()}`);
```

In the GIF, I click the "load more customers" button and you can see the fetch response start and finish immediately while the MSW log waits 2000ms to appear.

In my mocked endpoint, if I add the following lines, the fetch response is correctly delayed. 

```javascript
await new Promise((r) => {
  setTimeout(r, faker.random.number(2000));
});
```

Any idea what could be causing this issue? Thanks!

![msw-delay](https://user-images.githubusercontent.com/7675922/80894406-94083800-8c8f-11ea-8531-b5ce06c5eef1.gif)

**Comment by modularcoder** [comment]
+1 happens for me

**Comment by hehehai** [comment]
@mitchconquer Yes, it was a mistake.

This is because of the delay error in the mockserviceworker.js file.
You can start by modifying the documentation in the mockserviceworker.js file

![image](https://user-images.githubusercontent.com/12692552/80896034-a54b5600-8d1c-11ea-9b46-f326f311be57.png)

after modification
```js
setTimeout(
    resolve.bind(this, createResponse(clientMessage)),
    clientMessage.payload.delay,
)
```

We'll fix it as soon as possible

**Comment by modularcoder** [comment]
@hehehai that worked, thanks a lot!

**Comment by mitchconquer** [comment]
Awesome, thanks!!

**Comment by kettanaito** [comment]
@hehehai, thank you for the fix! Indeed, since the message structure change I've overlooked where the `delay` is stored. Your changes look good, will approve and release.

Note that by design you shouldn't modify the `mockServiceWorker.js` directly. Once the patch is released, the library will suggest you to replace it with `npx msw init` once more, which is the established way to propagate changes and fixes like this one to the worker file.

**Comment by kettanaito** [comment]
Released in [`0.15.5`](https://github.com/mswjs/msw/releases/tag/v0.15.5). Please update and run `npx msw init <PUBLIC_DIR>` once more for the changes in the worker file to propagate to your project automatically.

Thank you once more for raising this and for a quick fix.

**Comment by shubhamdhingra38** [comment]
To anybody still facing this issue, for me changing my host from `http://0.0.0.0:4321` to `http://localhost:4321` worked for me.
